### ISSUE TITLE
Touristic/cultural destinations and accommodation on non-buildings should have a homogeneous expression (meta-issue)

### ISSUE THREAD

**Issue by polarbearing** [gravitystorm/openstreetmap-carto#1624]
I found that touristic+cultural destinations as well as touristic accommodation, when mapped as an area, and not being a building, are represented quite heterogeneously. However it would make the map more readable if they came with the same visual language.

This analysis focuses on 'dedicated targets', from museums to theme_parks; as opposed to daily-use and pitch-like ones (park, pitch, sports_centre, playground, recreation_ground, golf_course...)

Here is some code review how they are represented currently. 
(table might scroll horizontally)

Key | area (colour) | label (colour) | icon (colour) | issues | comment
--- | --- | --- | --- | --- | ---
*Destinations* | 
tourism=museum | *none* | fixed (amenity-brown) | museum.16.svg (amenity-brown) | | (museum parks, open-air exhibitions) 
tourism=zoo | boundary (tourism) | scales (tourism) | *none* | 
tourism=attraction | *removed* (was pink) | scales (#660033) | *none*  | #1257 | (often used for specific areas within a larger facility, or on its own) 
tourism=theme_park | boundary (tourism) | scales (tourism) | *none* | 
leisure=nature_reserve, boundary=national_park | admin-boundary (green) | scales (dark park) + boundary (green) | *none*  | #603
leisure=marina | boundary (blue) | fixed (marina-text) | 
leisure=water_park | *none* | fixed (dark park) | water_park.16.svg (amenity-brown) | #1490
historic=archaeological_site | *none* | *none* | archaeological_site.16.svg (amenity-brown) | #732 
leisure=beach_resort | *none* 
*Accommodation* | 
camp_site | fill (campsite) | fixed (#0066ff) | camping.16.svg (transportation-icon) | #1485 #1020 #1355
caravan_site | fill (campsite) | scalable (dark campsite) | caravan_park.16.svg (transportation-icon)
hostel | *none* | fixed (#0066ff) | hostel.16.svg (transportation-icon) || (area typically summer-camp style bungalow villages)
hotel  |  *none* | fixed (#0066ff) | hotel.16.svg (transportation-icon) | #775 | (resorts)
motel |  *none* | fixed (#0066ff) | motel.16.svg (transportation-icon) || (often arranged as area with separate buildings)
resort | *none* |||| (only 312 uses, probably hotel more popular)
summer_camp | *none* |||| (only 100 uses)

Current colours: campsite = #def6c0; tourism = amenity-brown = #734a08; transportation-icon = #0092da;

When it comes to colours, some countries have brown traffic signs for the touristic/cultural destinations, so that might fit most if not all of them, and I have seen green signs for local accommodation as well. 

I'll come up later with more considerations, but I have some preference for variations of the boundary style as currently used in Zoo and Theme_park, for most of them, maybe all destinations brown and some suitable colour for all accommodation.

**Comment by matkoniecz** [comment]
Note that object on the list belong to multiple categories. For example leisure=nature_reserve, boundary=national_park are also natural features and green fits well there.

**Comment by polarbearing** [comment]
> ... natural features and green fits well there.

Yes absolutely. I was not to propose to change that, the table was primarily to analyse what style we use for all these features. I tend to think that many of the sites above are humanly defined boundaries around a collection of objects, thus drawing the boundaries with no fill represents that well, leaving the fill colour for the nature+building details within them. Example borrowed from #778.

![](https://cloud.githubusercontent.com/assets/1190866/7672098/02cfb162-fc9b-11e4-89cc-e3034fa47be6.png)

**Comment by dieterdreist** [comment]
sent from a phone

> Am 30.06.2015 um 07:04 schrieb Mateusz Konieczny <notifications@github.com>:
> 
> For example leisure=nature_reserve, boundary=national_park are also natural features and green fits well there.


these are both legal features and any colour fits ;-)

**Comment by dieterdreist** [comment]
this is an interesting comparison table but it would be even better if the area column would be split into fill (solid/hatch/pattern) and outline. Outline might also be positioned different (outside, inside and centre)

**Comment by mboeringa** [comment]
> **I tend to think that many of the sites above are humanly defined boundaries around a collection of objects, thus drawing the boundaries with no fill represents that well**, leaving the fill colour for the nature+building details within them.

+1 

I fully concur with these observations and have come to the same conclusions regarding many of these tagged objects while developing my own renderer, and implemented them as such, either with outlines, open hatches or open patterns, but no solid fills.

**Comment by polarbearing** [comment]
I have added "societal amenities" to the comparison in the OP, such as educational/social/community, which are important for daily life as opposed to occasional travel destinations.

Following the discussion about the usage of outlines, I think that general landuse (often comprising multiple entities) should keep the fill colours, while singular entities, that have "humanly defined boundaries", should get an outline for their campus.

**Comment by kocio-pl** [comment]
I think the outlines can be even more useful than the fill colors - [this hospital area](http://www.openstreetmap.org/way/66266360) is mapped with such a lot of details, that yellow background is barely visible.

**Comment by kocio-pl** [comment]
This issue seems to be very broad and hard to fix in general, but I have specific proposition.

Since sport and recreation areas now use leisure green, we have hospitals/clinics and schools/universities again using common yellow background, but since we called it "societal amenities", I guess there are two more types which clearly belong to this namespace:
- amenity=community_centre - ~36k line/relation uses
- amenity=social_facility - ~21k line/relation uses

What do you think about it?

**Comment by polarbearing** [comment]
This thread here is called meta-issue since it is an instrument to collect and coordinate. The table in the top post is meant as a quick reference. 

Yes I am in favour of adding amenity=community_centre and amenity=social_facility to societal amenities yellow, see #2087 and [#506 comment](https://github.com/gravitystorm/openstreetmap-carto/issues/506#issuecomment-107402429).

**Comment by kocio-pl** [comment]
Thanks! Referenced now.
